conda_build_config.yaml

```diff
@@ -1,11 +1,17 @@
 python:
-  - 3.7.3
+  - 3.7
 
-liblapack:
-  - 3.8.0
+# liblapack:
+#   - 3.8.0
+
+# _openmp_mutex:
+#   - 4.5 1_gnu
+
+# openblas:
+#   - 0.3.5.*
```

Comment:
Removing these requirements and pinned version works locally? I see builds pass, so maybe enough.